src/api.test.js

```diff
@@ -40,3 +40,29 @@ describe("getConferenceList", () => {
   });
 });
 
+//test data for getConferencesById
+describe("getConferenceById", () => {
+  const mockData = {
+    data:
+    {
+      id: 1,
+      name: "Festival of Marketing",
+      dateTime: "2019-11-12T12:34:11Z",
+      city: "London",
+      description:
+        "From Festivalofmarketing.com: The Festival of Marketing is a unique experience where ambitious marketers can discover, learn, celebrate and shape the future together. As the largest global event dedicated to brand marketers, the Festival reflects the very nature of ...",
+      topic: "Marketing"
+    }
+  };
+
+  //test to return conference by id
+  it("returns conference by id", async () => {
+    expect.assertions(1);
+
+    axios.get.mockResolvedValue(mockData);
```

Comment:
This test is only really testing whether the mockResolvedValue works, which we can assume it does. I'm not totally sure we need a test for this file